packages/common/src/utils/testing/testing.tsx

```diff
@@ -1,19 +1,27 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect } from 'react';
```

Comment:
I took the testing provider and put it here.. i'm not really sure why, but it felt better here 🤷 

(though this file needs to be moved around also 😁 )